modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -67,6 +68,8 @@ struct ParamDesc {
     std::vector<bool> normalize; //!< Vector of bool values that enabled or disabled normalize of input data.
 
     std::vector<std::string> names_to_remap; //!< Names of output layers that will be processed in PostProc function.
+
+    bool is_generic;
```

Comment:
Please add documentation for this flag.

Does it really needed? (I see only `false` initial value)

Reply:
I believe it may be related to "generic infer" (with no network type defined), but let's @TolyaTalamanov to confirm

Reply:
Yes, it's needed, because only `generic` infer can be exposed to python